### PR TITLE
[next][tests] skip some next.js tests

### PR DESCRIPTION
**Pull request by EndangeredMassa**
These tests are consistently failing:

- https://github.com/vercel/vercel/actions/runs/11236739944/job/31240688177?pr=12242#step:8:1074
- https://github.com/vercel/vercel/actions/runs/11236739944/job/31240690783?pr=12242#step:8:354

### CODE REVIEW

File: .changeset/metal-suns-heal.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@vercel/next": patch
+---
+
+[next][tests] skip some next.js tests
```
Comment (ijjk) [inline .changeset/metal-suns-heal.md:5]: Shouldn't this be empty if it's just tests?

**Review by ijjk — COMMENTED** [review]
This looks like easy fix, opening separate PR

File: .changeset/metal-suns-heal.md (hunk 1 above)
Comment (EndangeredMassa) [reply .changeset/metal-suns-heal.md:5]: ah yes; if we do skip these, I'll remove that

**Comment by EndangeredMassa** [comment]
Closing in favor of: https://github.com/vercel/vercel/pull/12247

**Pull request closed without merging**